### PR TITLE
feat: migrate back from cmd to git root main

### PR DESCRIPTION
**Pull request by viktorvoltaire**

### CODE REVIEW

**Review by Edholm — APPROVED** [review]
Is this due to ease-of-use or ?

**Comment by odsod** [comment]
@Edholm Yes, it creates a bit more "productified" CLI interface when it's one CLI with subcommands - and as an added bonus we can move the example(s) up to the top level!

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@
 sagefile := .sage/tools/bin/sagefile
 
 $(sagefile): .sage/go.mod $(shell find .sage/.. -type f -name '*.go')
-	@cd .sage && go run ../cmd/build
+	@cd .sage && go run ../main.go run
```
Comment (odsod) [inline Makefile:9]: did we rename it to `run` instead of `build`?

**Review by odsod — APPROVED** [review]

**Review by odsod — APPROVED** [review]
Wow!

**Pull request merged**